cli/main.rs

```diff
@@ -129,6 +134,31 @@ async fn run(cli: Cli) -> Result {
                 }
             }
         }
+        Cli::Devices { cmd } => {
+            let config = config::load(CONF_PATH)?;
+            let client = client::Client::new(config)?;
+            match cmd {
+                DevicesCmd::All => {
+                    #[derive(Deserialize, Serialize)]
+                    struct Output {
+                        devices: Vec<DetailedDevice>,
+                    }
+                    let output = Output {
+                        devices: client.get_detailed_devices().await?,
+                    };
+                    println!("[");
```

Comment:
Sorry, I didn't receive/notice the review request. Just a FYI: if this command proves to be slow for large lists, locking stdout and writing to it will likely help.

Reply:
Thanks for the reminder; addressed in [PR #22](https://github.com/helium/helium-console-cli/pull/22)